### DISCUSSION TITLE
Blobfuse behaviour when internet Disconnects and Connects Again

### DISCUSSION THREAD

**Discussion by francoisdutoit** [Azure/azure-storage-fuse#932] [Q&A] [answered]
Hi, 

I was wondering - how does Blobfuse behave in cases where the internet connection drops and comes back online - considering the following two scenarios.

1.) Blobfuse mounts on startup - but internet connectivity is lost for a couple of hours and then comes back on.
2.) Blobfuse can't mount on startup due to lost internet connectivity - but internet connectivity is restored some time after startup.

How can we make sure Blobfuse mounts again after both cases mentioned and all files are uploaded?

**Comment by bhansalivikas** [comment, marked as answer]
I doubt there is any automated way to remount. If you are using /etc/fstab to mount blobfuse on startup, post connectivity is restored you can use "mount -av" to remount all failed mounts. Another option is to just mount manually once network is restored. 
One possible way I could suggest is, if the network always takes some time to come up then create a script that sleeps for few minutes initially and then mounts blobfuse and just kick start this script on boot.
